config/locales/metadata/metadata-pt.yml

```diff
@@ -17,4 +17,4 @@ pt:
       template_file_anchor: este arquivo de modelo
       upload: Envio
       upload_metadata: Carregar metadados
-      upload_metadata_description: Para atualizar os metadados de várias obras desta coleção, carregue uma planilha com uma linha por obra. Faça o download do %{link} para começar.
+      upload_metadata_description: Para atualizar os metadados de várias obras desta coleção, carregue uma planilha com uma linha por obra. <br> Faça o download do %{link} para começar.
```

Comment:
This does not seem to contain the note about stars.

Reply:
I believe I used the google_translate api you provided here. Can you please let me know the correct translation here? (Given that the translate api key seems to have translated this wrong?)

Reply:
Strange -- this looks fine now.  My apologies.